Python-packages/covidcast-py/docs/changelog.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+v0.1.6, May _, 2023
+-------------------
+
+- New feature: This package now supports queries that require an API key. Call
+  :py:func:`covidcast.use_api_key` before running fetch functions to
+  authenticate your requests.
+
```

Comment:
Not sure if this belongs in this PR or if it should wait for release

Reply:
I believe it'll only get deployed to the website when you commit the re-rendered HTML docs, right? So it should be fine to include here, then update when we make the PyPI release